tests/api/test_core_coord_translation_gs.cpp

```diff
@@ -52,8 +52,10 @@ TEST(CoordinateManager, CoordinateManagerGrayskullTopLeftCore) {
 // We expect that the top left core will have virtual and physical coordinates (1, 1) and (1, 2) for
 // the logical coordinates if the first row is harvested.
 TEST(CoordinateManager, CoordinateManagerGrayskullTopLeftCoreHarvesting) {
+    // This is targeting first row of Tensix cores on NOC layout.
+    const size_t harvesting_mask = (1 << 8);
```

Comment:
Do you think it might make sense to have inverse of HARVESTING_NOC_LOCATIONS? For blackhole that would be:
`static const std::vector<uint32_t> LOGICAL_HARVESTING_LAYOUT = {0, 2, 4, 6, 8, 10, 12, 13, 11, 9, 7, 5, 3, 1};`

Then
`harvesting_mask = (1 << LOGICAL_HARVESTING_LAYOUT[0])`
and this is the same for all arch

Reply:
makes sense, will do it

Reply:
I am jos not sure whether we would like to expose this through the *_implementation files. Thoughts?